src/cs/usfca/edu/histfavcheckout/model/ConfigRequest.java

```diff
@@ -0,0 +1,11 @@
+package cs.usfca.edu.histfavcheckout.model;
+
+import java.io.Serializable;
+
+public class ConfigRequest implements Serializable {
+	private boolean ignoreExternalAPIs;
+	
+	public boolean getIgnoreExternalAPIs() {
+		return ignoreExternalAPIs;
```

Comment:
I am not sure how is spring able to set ignoreExternalAPIs without a setter method.

Reply:
The values are [set](https://github.com/usfcs-perf-eng-s20/heroku-project-favs/pull/11/files#diff-0fae860ae0c1e35be827d26bc884c318R291) in the global config object declared [here](https://github.com/usfcs-perf-eng-s20/heroku-project-favs/pull/11/files#diff-af16990c66da3e10519e89df67af4655R40). So we can just read the value of "ignoreExternalAPIs" from that object